app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -401,7 +358,7 @@ class OtherUserProfileScreenViewModel @Inject constructor(
                     showInfoMessage(InfoMessageType.UnblockingUserOperationError)
                 }
             }
-            unblockUserDialogState = null
+            requestInProgress = false
```

Comment:
Hmm in the future I want to move that kind of functionality to some BaseViewModels, can I leave it as it is for now? This will change when I split that functionality to smaller view models 😄

Reply:
yeah, make sence